provider-ui/.gitignore

```diff
@@ -1,6 +1,18 @@
-# Cypress Tests Files
-cypress/videos
-cypress/screenshots
```

Comment:
This seems like something we would want to ignore...

Reply:
Fixed